### DISCUSSION TITLE
monorepo support for selective builds

### DISCUSSION THREAD

**Discussion by the-leonat** [coollabsio/coolify#1898] [Feature Requests]
i have frontend/backend application in one repository and created separate application in coolify with the use base-dir setting.
if i commit changes to the frontend both applications are redeployed. is there a way to redeploy only the application which had changes?

**Comment by andrasbacsai** [comment]
I soon add support for include paths with globs, so you can define a gitignore like paths that will trigger a deployment.

**Reply by andrasbacsai** [reply]
I already added this. It is called `Watch Paths`.

**Reply by boredland** [reply]
Could this also work for docker-compose deployments? I basically only want to trigger a deployment if the compose-file changed..
